tests/registry_disk_test.go

```diff
@@ -168,4 +171,34 @@ var _ = Describe("ContainerDisk", func() {
 			})
 		})
 	})
+
+	Describe("Starting with virtio-win", func() {
+		Context("with virtio-win as secondary disk", func() {
+			It("should boot and have the virtio as sata CDROM", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
```

Comment:
Should this test be in our windows test section rather than the container disk?  From what I gather this test is attempting to prove we can expose virtio drivers on a container disk to a windows host. Is that a correct interpretation?

Reply:
I agree with you, there should be windows test for this.
Unfortunately, currently there is no easy way for providing automated test during windows install. Even if this would be possible, fully testing installing windows and than checking if the driver is there, would possible take hour+.

Therefore I created test to check if the drivers can be mounted as SATA CD-ROM and check whether the files are there. This should be sufficient to guarantee that windows will see then too.